submissions/morenko/task-frogger/js/app.js

```diff
@@ -0,0 +1,177 @@
+const countRows = 6;
+const countColumns = 5;
+
+const tileHight = 83;
+const tileWidth = 105;
+
+class Position {
+    startX;
+    startY;
+
+    x;
+    y;
+
+    minY;
+    minX;
+    maxY;
+    maxX;
+    
+    constructor( row, col ) {
+        
+        this.startX = (tileWidth * col) - (tileWidth);
+        this.startY = (tileHight * row) - (tileHight / 2);
+        this.x = this.startX;
+        this.y = this.startY;
+
+        this.maxY = tileHight * (countRows - 1);
+        this.maxX = tileWidth * (countColumns - 1);
+        this.minY = tileHight * (row - 1) - this.startY;
+        this.minX = 0;
+    }
+
+    reset() {
+        this.x = this.startX;
+        this.y = this.startY;
+        
+    }
+}
+
+class Enemy {
+
+    speed = (Math.floor(Math.random(4) * (500 - 100 + 1)) + 100);
+    sprite = 'images/Rock.png';
+
+    constructor ( PositionEnemy ) {
```

Comment:
We start names of values with a capital letter only when describing a class. `PositionEnemy` is just an object here.